hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -451,7 +451,14 @@ public boolean isMetricsEnabled() {
 
     /**
      * Returns whether metrics should be stored in the cluster after the job
-     * completes. If enabled, metrics can be retrieved by calling {@link Job#getMetrics()}.
+     * completes. If enabled, metrics can be retrieved for all non-running
+     * jobs (have completed successfully, have failed, have been cancelled
```

Comment:
1. We normally document the set-methods and the get-methods refer to those. The reason is that the user mostly uses only the set-methods, the get-methods are used by Jet internally.

2. it doesn't apply to "all non-running jobs" but only to this job, when it's not running.